eks.tf

```diff
@@ -27,7 +27,7 @@ module "eks" {
   # copy of the pending PRs
   source = "terraform-aws-modules/eks/aws"
   # version of the eks module to use
-  version = "8.1.0"
+  version = "13.2.1"
```

Comment:
Make sure to run `terraform fmt` on this repo during the upgrade. There is already a `pre-commit` hook to do this, so make sure you've run `pre-commit install` in the git root to make use of that.